src/components/legend/attributes.js

```diff
@@ -163,9 +163,23 @@ module.exports = {
         max: 3,
```

Comment:
@alexcjohnson 
Should these `min` and `max` values be applied for *container* or do we want to restrict them?

Reply:
When applied to container, I've separately hard-coded these values to be within [0, 1]. What's the best way to have that reflected in the attribute definition here?

Reply:
We should possibly 
1. remove `min` and `max` from the attribute itself. 
2. describe `min` and `max` in the description depending on being `container` or `paper` referenced.
3. at the supply defaults step, set any out bound values to `dflt` depending on orientation, etc.
4. simplify plot step

Reply:
Makes sense. I can work on implementing this. @alexcjohnson do you agree that this would be the correct approach?

Reply:
Two things: (1) ensuring the logic is exactly the same as all other attributes for out-of-bounds values (do you clip to the nearest valid value, or discard and use the default, for example), (2) the reference will not have misleading info in it, ie [legend.x](https://plotly.com/javascript/reference/layout/#layout-legend-x) says:
> _Type_: number between or equal to -2 and 3

which will no longer be true in all cases. Specifying the bounds in the description instead lets you be more precise.

Reply:
Makes sense! Have applied this change [here](https://github.com/plotly/plotly.js/pull/6589/commits/a01361cb6b3fdda202ce5ae17606d5dfd278784c)